clippy_dev/src/fmt.rs

```diff
@@ -176,22 +173,3 @@ fn rustfmt(context: &FmtContext, path: &Path) -> Result<bool, CliError> {
     }
     Ok(success)
 }
-
-fn project_root() -> Result<PathBuf, CliError> {
```

Comment:
This function is also copy pasted in 
https://github.com/rust-lang/rust-clippy/blob/bbef531518a3b4977a9c1f40808f950514b400fc/clippy_dev/src/new_lint.rs#L85-L101